app/comparator.py

```
@@ -0,0 +1,134 @@
+        
+        if not diff:
+            return "# No Changes\nNo differences found in OpenAPI specs."
+        
+        # Initialize categories
+        categories = {
+            "API Changes": [],
+            "Production Changes": []
+        }
+        
+        # Categorize changes
+        for change_type, changes in diff.items():
```

Ruff (0.8.2)

Loop control variable `change_type` not used within loop body

Rename unused `change_type` to `_change_type`

(B007)